requirements.txt

```diff
@@ -0,0 +1,16 @@
+appdirs~=1.4.3
+asgiref~=3.2.7
+attrs~=19.3.0
+beautifulsoup4~=4.9.0
+black~=19.10b0
+click~=7.1.2
+Django~=3.0.6
+djangorestframework~=3.11.0
+pathspec~=0.8.0
+pyparsing~=3.0.0a1
```

Comment:
Can we downgrade this to 2.4.7? I didn't realize I installed an alpha version, apologies.

Reply:
https://github.com/UWmathNEWS/slugline-api/pull/11/commits/d9f4c0561b68b87e354f5921ef3f6b6a6d575d8f

Thy will be done.